File: backend/src/main/java/loafer/domain/Topic.java
package loafer.domain;

import lombok.Data;

import javax.persistence.Entity;
import javax.persistence.Table;
import java.io.Serializable;

/**
 * Created by wuyuexin on 2017/6/3.
 */
@Data
@Entity
@Table(name = "topic")
public class Topic extends AbstractAuditingModel implements Serializable{

    private String name;

    private String description;
}

File: loafer-webapp/src/main/webapp/src/app/models/Topic.ts
export class Topic {
  private id: number;
  private name: string;
  private description: string;
  private image: string;

  constructor(id: number, name: string, description: string, image: string) {
    this.id = id;
    this.name = name;
    this.description = description;
    this.image = image;
  }
}
